pkg/handlers/mount.go

```diff
@@ -110,11 +119,69 @@ func Mount(allowedFilesystems map[string]struct{}) registry.HandlerFunc {
 			return registry.HandlerResultContinue()
 		}
 
+		var options string
+		if req.Data.Args[4] != 0/* NULL */ && filesystem != "proc" && filesystem != "sysfs" {
```

Comment:
Just curious, why only if this is != 0?

Reply:
Options is optional, in which case the argument is 0, as in char* NULL.

Reply:
Ouch, makes sense